scvi/module/_jaxvae.py

```diff
@@ -88,8 +91,9 @@ def setup(self):
             "disp", lambda rng, shape: jax.random.normal(rng, shape), (self.n_input, 1)
         )
 
-    def __call__(self, z: jnp.ndarray, batch: jnp.ndarray, training: bool = False):
+    def __call__(self, z: jnp.ndarray, batch: jnp.ndarray, training=False):
```

Comment:
missing typing

Reply:
also shouldnt this default to None?

Reply:
should probably have a tests for both cases of the merge param

Reply:
Done